tests/sqllogictests/suites/query/02_function/02_0012_function_datetimes

```diff
@@ -893,3 +893,38 @@ select * from t order by b
 statement ok
 drop table t
 
+query T
+select date_format('2022-02-02', '精彩的%Y年，美丽的%mmonth,激动の%dd');
+----
+精彩的2022年，美丽的02month,激动の02d
+
+query T
+select str_to_date('精彩的2022年，美丽的02month,激动の02d', '精彩的%Y年，美丽的%mmonth,激动の%dd');
```

Comment:
The name str_to_date is confusing here, string to date is a parse a string to a date data type.

Reply:
In mysql doc: https://dev.mysql.com/doc/refman/8.0/en/date-and-time-functions.html#function_str-to-date